@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/core/fn_coreControlHandle.sqf

```diff
@@ -125,6 +120,12 @@ if (_value) then {
         case "fza_ah64_forceTrimHoldModeSwitch_left": {
             [_heli] call fza_sfmplus_fnc_fmcAttitudeHoldEnable;
         };
+        case "launchCM": {
```

Comment:
fza_ah64_chaff

Reply:
This is an arma default Output for the vanilla Keybind, cannot be changed

Reply:
Gotcha